Description: Upstream changes introduced in version 0.4.1-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 surf (0.4.1-3) unstable; urgency=low
 .
   * New maintainer
 .
 The person named in the Author field signed this changelog entry.
Author: Kai Hendry <hendry@iki.fi>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ surf-0.4.1/config.h
@@ -0,0 +1,40 @@
+/* modifier 0 means no modifier */
+static char *useragent      = "Surf/"VERSION" (X11; U; Unix; en-US) AppleWebKit/531.2+ Compatible (Safari)";
+static char *progress       = "#FF0000";
+static char *progress_trust = "#00FF00";
+static char *stylefile      = ".surf/style.css";
+static char *scriptfile     = ".surf/script.js";
+static char *cookiefile     = ".surf/cookies.txt";
+static time_t sessiontime   = 3600;
+#define NOBACKGROUND 0
+
+#define SETPROP(p, q)     { .v = (char *[]){ "/bin/sh", "-c", \
+	"prop=\"`xprop -id $2 $0 | cut -d '\"' -f 2 | dmenu`\" &&" \
+	"xprop -id $2 -f $1 8s -set $1 \"$prop\"", \
+	p, q, winid, NULL } }
+#define DOWNLOAD(d) { \
+	.v = (char *[]){ "/bin/sh", "-c", \
+	"xterm -e \"wget --load-cookies ~/.surf/cookies.txt '$0';\"", \
+	d, NULL } }
+#define MODKEY GDK_CONTROL_MASK
+static Key keys[] = {
+    /* modifier	            keyval      function    arg             Focus */
+    { MODKEY|GDK_SHIFT_MASK,GDK_r,      reload,     { .b = TRUE } },
+    { MODKEY,               GDK_r,      reload,     { .b = FALSE } },
+    { MODKEY|GDK_SHIFT_MASK,GDK_p,      print,      { 0 } },
+    { MODKEY,               GDK_p,      clipboard,  { .b = TRUE } },
+    { MODKEY,               GDK_y,      clipboard,  { .b = FALSE } },
+    { MODKEY|GDK_SHIFT_MASK,GDK_j,      zoom,       { .i = -1 } },
+    { MODKEY|GDK_SHIFT_MASK,GDK_k,      zoom,       { .i = +1 } },
+    { MODKEY|GDK_SHIFT_MASK,GDK_i,      zoom,       { .i = 0  } },
+    { MODKEY,               GDK_l,      navigate,   { .i = +1 } },
+    { MODKEY,               GDK_h,      navigate,   { .i = -1 } },
+    { MODKEY,               GDK_j,      scroll,     { .i = +1 } },
+    { MODKEY,               GDK_k,      scroll,     { .i = -1 } },
+    { 0,                    GDK_Escape, stop,       { 0 } },
+    { MODKEY,               GDK_o,      source,     { 0 } },
+    { MODKEY,               GDK_g,      spawn,      SETPROP("_SURF_URI", "_SURF_GO") },
+    { MODKEY,               GDK_slash,  spawn,      SETPROP("_SURF_FIND", "_SURF_FIND") },
+    { MODKEY,               GDK_n,      find,       { .b = TRUE } },
+    { MODKEY|GDK_SHIFT_MASK,GDK_n,      find,       { .b = FALSE } },
+};
